experimental/devices/nrzled/doc.go

```diff
@@ -8,6 +8,15 @@
 //
 // Note that some ICs are 7 bits with the least significant bit ignored, others
 // are using a real 8 bits PWM. The PWM frequency varies across ICs.
+// User can select a driver implementation using a memory-mapped GPIO line
+// or leverage a SPI connection's MISO pin.
+//
+// The SPI implementation is sensitive to variations in SPI clock speed.
+// On the Raspberry Pi, you will need to add `core_freq=250`
+// to /boot/config.txt to prevent glitching.
+//
+// You may also need to increase your SPI buffer size to 12*num_pixels+3, or just max it out
+// with `spidev.bufsize=65536`. That should allopw you to buffer over 5400 Neopixels.
```

Comment:
[just FYI, not action]

I have a 256 LEDs board at home that I'm very eager to try out.

I also want to try out via an FT232H but hadn't had the chance to complete the driver yet; https://periph.io/x/extra/hostextra/d2xx